README.md

```diff
@@ -19,6 +19,8 @@ The following procedure describes how to deploy the `ExternalDNS` Operator for A
    oc create ns external-dns
    oc apply -f config/samples/extra-roles.yaml
    ```
+   **Note**:  For OpenShift clusters use `config/samples/extra-roles-openshift.yaml` instead.
```

Comment:
Why the extra roles are in `config/samples/` not `config/rbac/` ?

Reply:
Oh I was just afraid they would get confused with ones that go under the bundle. But I can move them there.

Reply:
We have full control over what's going in the bundle thanks to `olm-manifests` Makefile target. We can simply not copy the extra role files in in the bundle.

Reply:
Sorry, my bad, you are right there was an `if` for the route source. Well, yes, may be it makes sense to separate them. But just for a note: keeping them in one file won't harm.